docs/content/integration.md

```diff
@@ -9,14 +9,21 @@ OPA exposes domain-agnostic APIs that your service can call to manage and
 enforce policies. Read this page if you want to integrate an application,
 service, or tool with OPA.
 
+When integrating with OPA there are two interfaces to consider:
+* **evaluation**: the interface for asking for policy decisions.  This is primarily
+* **management**: the interface for deploying policies, understanding status, uploading logs, and so on
```

Comment:
Nit: the bullet points did not render correctly.

Reply:
Fixed.